### PR TITLE
TASK: Adjust Flow to work with PHP 7.2

### PR DESCRIPTION
**Pull request by kdambekalns**
This enables PHP 7.2 testing on Travis CI, fixes a few unit tests that broke and adjust one actual code error.

### CODE REVIEW

File: Neos.FluidAdaptor/Tests/Unit/Core/ViewHelper/AbstractViewHelperTest.php [hunk 1]
```diff
@@ -88,6 +88,8 @@ public function setUp()
      */
     public function argumentsCanBeRegistered()
     {
+        $this->mockReflectionService->expects($this->any())->method('getMethodParameters')->will($this->returnValue([]));
```
Comment (albe) [inline Neos.FluidAdaptor/Tests/Unit/Core/ViewHelper/AbstractViewHelperTest.php]: Should this maybe be put into `setUp()`?

File: Neos.Flow/Tests/Unit/Mvc/Routing/IdentityRoutePartTest.php [hunk 2]
```diff
@@ -139,6 +139,8 @@ public function matchValueSetsTheIdentifierOfTheObjectPathMappingAndReturnsTrueI
      */
     public function matchValueSetsTheRouteValueToTheUrlDecodedPathSegmentIfNoUriPatternIsSpecified()
     {
+        $this->mockClassSchema->expects($this->any())->method('getIdentityProperties')->will($this->returnValue([]));
```
Comment (albe) [inline Neos.Flow/Tests/Unit/Mvc/Routing/IdentityRoutePartTest.php]: Should this maybe be put into `setUp()`?

File: Neos.Flow/Tests/Unit/Persistence/Generic/SessionTest.php [hunk 3]
```diff
@@ -583,8 +583,11 @@ public function getIdentifierByObjectReturnsUuidForObjectBeingAOPProxy()
         $knownObject = $this->createMock(ProxyInterface::class);
         $knownObject->Persistence_Object_Identifier = 'fakeUuid';
 
+        $mockReflectionService = $this->getMockBuilder(ReflectionService::class)->setMethods(['getPropertyNamesByTag'])->getMock();
```
Comment (albe) [inline Neos.Flow/Tests/Unit/Persistence/Generic/SessionTest.php]: Maybe a candidate for setting up generally in `setUp()`?

**Comment by kdambekalns** [comment]
I just added changes to the tests that needed fixing, not sure putting some of the changes in `setUp()` would work with all other tests…

**Pull request merged**